labcontrol/db/tests/test_process.py

```diff
@@ -1493,6 +1490,12 @@ def test_bcl_scrub_name(self):
         self.assertEqual(SequencingProcess._bcl_scrub_name('test-1'), 'test-1')
         self.assertEqual(SequencingProcess._bcl_scrub_name('test_1'), 'test_1')
 
+    def test__folder_scrub_name(self):
+        input= "Ogden  Bogden-Meade*,_Pat O'Brien_1"
```

Comment:
Minor, but I think flake8 may twitch here with the missing space prior to the `=`

Reply:
Thanks!  If you don't mind, I'll take care of this in the next PR ... :D